docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Yeoman Gulp-Angular Generator documentation
```

Comment:
It looks like this file is also leftover from the original template, can it be deleted?  I suspect the entire `docs` folder can actually be deleted.